inc/class-wpseo-utils.php

```diff
@@ -195,34 +219,35 @@ public static function trim_recursive( $value ) {
 	 * @return string
 	 */
 	public static function translate_score( $val, $css_value = true ) {
-		if ( $val > 10 ) {
```

Comment:
Ok, so I guess this function expects a value between 0 and 9 OR it divides by 10 and rounds to get a value between 0 an 9. Everywhere where this function is now fed with a value between 0 and 9 it will break (like in the page analysis), since our seo_score translation is now based on a scale of 1 to 100.